.github/workflows/update.yml

```diff
@@ -0,0 +1,56 @@
+name: Update CKEditor 4 submodule
+
+on:
+  schedule:
+  - cron: "0 0,10,16 * * *"
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        branch: [master, major]
+
+    steps:
+      - name: Fetch changes
+        # https://github.com/marketplace/actions/checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.branch }}
+
+      - name: Prepare update
+        run: |
+          branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Can't we reuse `matrix.branch` here? GHA have [special syntax](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsenv) for it.

Reply:
Somehow missed that TBH. Will use it if necessary after switching to ready-made workflow mentioned in https://github.com/ckeditor/ckeditor4-presets/pull/34#issuecomment-614048429.

Reply:
Funny, I already used it few lines above with `ref: ${{ matrix.branch }}` :see\_no\_evil: